car.py

```diff
@@ -0,0 +1,32 @@
+"""Extract and validate car names."""
+import re
+from parse import parse
+
+def is_valid(car_name):
+    """Take in a string containing a car name. Return True if it is
+    valid name. Otherwise, return False.
+    """
+    if re.search(r"[\w.-]+", car_name):
+        return True
+    return False
+
+def are_valid(car_names):
+    """Take in a list of car names. Return True if every car in the list is
+    valid. Otherwise, return False.
+    """
+    for each_car in car_names:
```

Comment:
reuse your is_valid method here on the list rather than reimplement the regex each time